corehq/apps/userreports/templates/userreports/expression_debugger.html

```diff
@@ -16,10 +16,22 @@ <h1>{% trans "UCR Expression Debugger" %}</h1>
     <div class="form-group">
       <label for="doc_type" class="col-sm-2 control-label">{% trans "Document Type" %}</label>
       <div class="col-sm-3">
-        <select id="doc_type" name="doc_type" class="form-control " data-bind="value: documentType">
-          <option value="case">{% trans "Case" %}</option>
-          <option value="form">{% trans "Form" %}</option>
-        </select>
+        <select-toggle data-apply-bindings="false"
```

Comment:
For my knowledge, Can you share how `select-toggle` is better here?
and what does `data-apply-bindings` does?

Reply:
The select toggle is better here because it's one fewer click and lets you see all options immediately. More info [here](https://www.commcarehq.org/styleguide/molecules/#molecules-selections)

`data-apply-bindings` determines whether this element should have bindings applied to it individually - it's `false` here because this element is inside a bigger container that has bindings applied to it. See https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/components.ko.js#L36